node/grpc/run.go

```diff
@@ -0,0 +1,81 @@
+package grpc
+
+import (
+	"errors"
+	"fmt"
+	"net"
+
+	pb "github.com/Layr-Labs/eigenda/api/grpc/node"
+	pbv2 "github.com/Layr-Labs/eigenda/api/grpc/node/v2"
+	"github.com/Layr-Labs/eigenda/common/healthcheck"
+	"github.com/Layr-Labs/eigenda/node"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+)
+
+const localhost = "0.0.0.0"
+
+func RunServers(serverV1 *Server, serverV2 *ServerV2, config *node.Config, logger logging.Logger) error {
+	if serverV1 == nil {
+		return errors.New("node V1 server is not configured")
+	}
+	if serverV2 == nil {
+		return errors.New("node V2 server is not configured")
+	}
+
+	go func() {
+		for {
+			addr := fmt.Sprintf("%s:%s", localhost, config.InternalDispersalPort)
+			listener, err := net.Listen("tcp", addr)
+			if err != nil {
+				logger.Fatalf("Could not start tcp listener: %v", err)
+			}
+
+			opt := grpc.MaxRecvMsgSize(60 * 1024 * 1024 * 1024) // 60 GiB
```

Comment:
Is this safe? Could an attacker force us to load 60gb into memory in order to force us to OOM?

Reply:
We don't have such protection on DA nodes right now (so reducing the max message size wouldn't make it any safer), and that's why they need to set up firewall and accept traffic from whitelisted origins for now.